django/core/management/base.py

```diff
@@ -277,6 +279,18 @@ def create_parser(self, prog_name, subcommand, **kwargs):
         Create and return the ``ArgumentParser`` which will be used to
         parse the arguments to this command.
         """
+
+        def add_base_argument(parser, *args, **kwargs):
+            """
+            Call the parser's add_argument() method, suppressing the help text
+            according to BaseCommand.suppressed_base_arguments.
+            """
+            for option_name in args:
+                if option_name[0] == '-' and option_name in self.suppressed_base_arguments:
```

Comment:
I think it would be clearer (and more correct?) if the initial lines were:

```python
for arg in args:
    if not arg.startswith('-'):
        break
    if arg in self.suppressed_base_arguments:
        ...
```

Also, is it a deliberate limitation that positional arguments (options not starting with `-`) aren't supported?

Reply:
That change has just been pushed to PR. Thank you.